codegen/src/main/resources/cordapp/cordapp_gradle.mustache

```diff
@@ -43,6 +56,7 @@ allprojects {
 
     repositories {
         mavenCentral()
+        mavenLocal()
```

Comment:
Is that required for some reason?

Reply:
Ah no, that was for my local testing, sorry, will remove it.